packages/server/package.json

```diff
@@ -50,6 +50,7 @@
     "@aws-sdk/client-s3": "^3.276.0",
     "@aws-sdk/lib-storage": "^3.100.0",
     "@aws-sdk/s3-request-presigner": "3.352.0",
+    "@bull-board/express": "^4.2.2",
```

Comment:
bull-board itself was already a non-dev dependency, might as well move this other one